service/api.go

```diff
@@ -783,6 +784,16 @@ func (s *gnoNativeService) AddressFromMnemonic(ctx context.Context, req *connect
 	return connect.NewResponse(&api_gen.AddressFromMnemonicResponse{Address: info.GetAddress().Bytes()}), nil
 }
 
+func (s *gnoNativeService) ValidateMnemonicWord(ctx context.Context, req *connect.Request[api_gen.ValidateMnemonicWordRequest]) (*connect.Response[api_gen.ValidateMnemonicWordResponse], error) {
+	valid := slices.Contains(bip39.EnglishWordList, req.Msg.Word)
```

Comment:
I think is better to use [IsMnemonicValid](https://github.com/gnolang/gno/blob/master/tm2/pkg/crypto/bip39/bip39.go#L224), what do you think?

Reply:
`IsMnemonicValid` is for a phrase. But we want `ValidateMnemonicWord` to check just one word.

Reply:
... In `IsMnemonicValid` it [checks each word of the phrase with `ReverseWordMap`](https://github.com/gnolang/gno/blob/e31395247f3aceb5e2e1dfbf550e5992b6ac14ff/tm2/pkg/crypto/bip39/bip39.go#L238) which is made from `EnglishWordList`. We don't need the map, so I just used the list.